doc/psud/PSU_daemon_design.md

```diff
@@ -88,14 +157,41 @@ Commands:
 
 The current output for "show platform psustatus" looks like:
 
+```
+admin@sonic:~$ show platform  psustatus
+PSU    Model          Serial        HW Rev      Voltage (V)    Current (A)    Power (W)  Status  LED
+-----  -------------  ------------  --------  -------------  -------------  -----------  ------- -----
+PSU 1  MTEF-PSF-AC-A  MT1629X14911  A3                12.08           5.19        62.62  WARNING green
+PSU 2  MTEF-PSF-AC-A  MT1629X14913  A3                12.01           4.38        52.50  OK      green
+```
+
+The field `Status` represents the status of the PSU, which can be the following:
+- `OK` which represents no alarm raised due to PSU power exceeding the threshold
+- `Not OK` which can be caused by:
+  - power is not good, which means the PSU is present but no power
```

Comment:
not sure how this can happen. can you please explain?

Reply:
`Not OK`? the PSU is present but the power cable is not plugged or power is down.